docs/api/platform/PowerManagement.md

```diff
@@ -81,6 +81,26 @@ UNLOCK: mbed_rtx_idle.cpp, ln: 131, lock count: 1
 
 <span class="notes">**Note:** Sleep tracing is a debug feature and should only be enabled during development cycle. Its heavy use of UART can affect the device performance.</span>
 
+By default, tickless mode is enabled for some targets that support it.
+
+You can override the default configuration in your application `mbed_app.json` and achieve further memory optimisation if tickless mode does not fit your application requirement as you may rely on SysTick always being present; then .
```

Comment:
I would add a sentence at the start to explain that there is a trade off between power saving and memory footprint. Something like:
"There is a small trade-off between power saving and memory footprint. When tickless mode is enabled, the memory footprint of your application is slightly bigger. Therefore if tickless mode does not fit your application requirement, for example you may rely on SysTick always being present, then you can disable tickless and achieve further memory optimisation. To do that, you can override the default configuration in your application `mbed_app.json` like this:"

Reply:
[Done](https://github.com/ARMmbed/mbed-os-5-docs/compare/d406b844efbabb14ebdb353bdb395bad023c4139..e6393fa5d1a2e54e35f14d29ab4cf25809a7f102).